### DISCUSSION TITLE
Training 8000 kHZ language identification model: Same language during inference

### DISCUSSION THREAD

**Discussion by kirillkoncha** [speechbrain/speechbrain#2049] [Q&A]
Hello!

I am training a model for language identification on 8kHZ audio. During the training, EER falls to 0.10. However, when I am testing model on the same validation set I used during training, the model output is the same language for all the audio files.

I encountered the similar [problem](https://github.com/speechbrain/speechbrain/issues/1516) during finetuning 16kHZ model. It was solved by shuffling batches. I made sure that batches are shuffled during current training.

It seems to me that the inference could be the problem. I wonder why the documentation of Encoder Classifier states that the audio must be 16000 kHZ?

> wavs : torch.Tensor
>             Batch of waveforms [batch, time, channels] or [batch, time]
>             depending on the model. Make sure the sample rate is fs=16000 Hz.

**Comment by kirillkoncha** [comment]
@TParcollet
